config/prometheus/collector_alerts.yaml

```
@@ -55,6 +55,36 @@ spec:
       labels:
         service: collector
         severity: Warning
+    - alert: ElasticsearchDeprecation
+      annotations:
+        message: "The OpenShift Elasticsearch Operator is deprecated and is planned to be removed in a future release. Red Hat provides bug fixes and support for this feature during the current release lifecycle, but this feature no longer receives enhancements. As an alternative to using the OpenShift Elasticsearch Operator to manage the default log storage, you can use the Loki Operator."
+        summary: "Detected Elasticsearch as the on cluster storage which is deprecated and will be removed in a future release."
```

The `on` tripped me up on first read, I think it needs a `-`:

```suggestion
        summary: "Detected Elasticsearch as the on-cluster storage which is deprecated and will be removed in a future release."
```

Also, isn't it "in-cluster"?

```suggestion
        summary: "Detected Elasticsearch as the in-cluster storage which is deprecated and will be removed in a future release."
```